CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Improve UX of Viewer List window and sync font with settings.
```

Comment:
I would suggest changing this to "Minor: Improved UI of the viewer list window"

Reply:
Done